homeassistant/components/vizio/config_flow.py

```diff
@@ -318,7 +319,7 @@ def __init__(self) -> None:
         return await self.async_step_user(user_input=import_config)
 
     async def async_step_zeroconf(
-        self, discovery_info: Dict[str, Any] = None
+        self, discovery_info: Optional[DiscoveryInfoType] = None
```

Comment:
The discovery info sent by the `zeroconf` integration is different than the ones sent by integrations to their own platforms. Should we still use this? Can't wait for `TypedDict` to be part of all our supported versions.

Reply:
`TypedDict` is backported via `typing-extensions` package.

https://pypi.org/project/typing-extensions/

It's one more dependency though.

Reply:
We could take the current lack of TypedDict as an opportunity to try and recognize+convert cases where a NamedTuple would do.